### PR TITLE
Add R116 release post

### PR DESCRIPTION
**Pull request by dilyand**

### CODE REVIEW

**Comment by dilyand** [comment]
@chuwy Could you please add a paragraph on the EER changes in section 4 and (if applicable) in Upgrading?

**Comment by chuwy** [comment]
Hey @dilyand! I think we can omit it because all changes are made to make RDB Loader R32 running and it is not released it - why someone would upgrade to EER 116 without Loader R32. I'll add a paragraph in R32 blog post though.

File: _posts/2019-09-06-snowplow-r116-madara-rider/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+The request paths we use have traditionally been intrinsically tied to how the collector (and downstream processes) work. With version `0.16.0` we are giving users of the Scala Stream Collector the option to map their own custom paths to the ones used under the hood. That way, users will be able, for example, to send requests to a path like `/sp/track`.
+
+The path can be changed when initialising the JavaScript tracker, as explained [here][post-path]. See below in the <a href="upgrading">Upgrading</a> section for details on how to configure the mappings in the collector.
+
+<h2 id="cache">3. Cache-Control headers</h2>
+
+To increase the likelihood that the Snowplow pixel will not be cached by a browser or proxy, we have added the following directives to the `Cache-Control` header of the collector's response: `no-cache`, `no-store`, `must-revalidate`.
+
+<h2 id="eer">4. EmrEtlRunner</h2>
+
```
Comment (yalisassoon) [inline _posts/2019-09-06-snowplow-r116-madara-rider/index.md]: Is there a missing section here?

Comment (yalisassoon) [reply _posts/2019-09-06-snowplow-r116-madara-rider/index.md]: Ah - just seen the conversation above - if we can omit that section let's remove and then we're good to publish - thanks again

File: _posts/2019-09-11-snowplow-r116-madara-rider/index.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+<br>
+Feradz [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0)
+
+<!--more-->
+
+<h2 id="cookies">1. Setting first-party cookies server-side against multiple domains on the same collector</h2>
+
+By default, no domain is specified for the cookie set by the Scala Stream Collector. That means the cookie is set against the full collector domain, for example `c.example.com`. That way, applications running elsewhere on `*.example.com` are not able to access it.
+
+In earlier versions, users could specify a `domain` in the collector configuration file to which to tie the cookie. For example, if set to `.example.com`, the cookie would have been accessible to other applications running on `*.example.com`. From version `0.16.0` on, to achieve that, users will have to use the new `fallbackDomain` setting in the configuration file.
```
Comment (LyubaGolovina) [inline _posts/2019-09-11-snowplow-r116-madara-rider/index.md]:
```suggestion
In earlier versions, users could specify a `domain` in the collector configuration file that they could tie the cookie to. For example, if set to `.example.com`, the cookie would have been accessible to other applications running on `*.example.com`. From version `0.16.0` on, to achieve that, users will have to use the new `fallbackDomain` setting in the configuration file.
```

File: _posts/2019-09-11-snowplow-r116-madara-rider/index.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+From version `0.16.0` we are providing users with ways to control several additional attributes for the `Set-Cookie` response header sent by the collector: `Secure`, `HttpOnly` and `SameSite`. See below in the <a href="upgrading">Upgrading</a> section for details on how to configure them.
+
+<h2 id="paths">3. Custom request paths</h2>
+
+We have been aware for some time that the Snowplow JavaScript tracker has been targeted by some ad-blocking providers, whose software blocks traffic to the paths used by the HTTP requests the tracker sends to the collector.
+
+We firmly believe that data subjects's rights must be protected and we are providing [tools and advice][gdpr] that makes that easy for Snowplow users. While we understand how ad-blockers might see a Snowplow tracker as a privacy invasion, we believe that is misguided. Snowplow enables event tracking and as such data flows through pipelines we've built. But we have no access to that data and no interest in it beyond ensuring that it is safely collected and stored in our users's own data warehouses. Blocking the Snowplow tracker is in effect preventing website owners from understanding their users and improving their service.
```
Comment (LyubaGolovina) [inline _posts/2019-09-11-snowplow-r116-madara-rider/index.md]:
```suggestion
We firmly believe that data subjects' rights must be protected and we are providing [tools and advice][gdpr] that makes that easy for Snowplow users. While we understand how ad-blockers might see a Snowplow tracker as a privacy invasion, we believe that is misguided. Snowplow enables event tracking and as such data flows through pipelines we've built. But we have no access to that data and no interest in it beyond ensuring that it is safely collected and stored in our users's own data warehouses. Blocking the Snowplow tracker is in effect preventing website owners from understanding their users and improving their service.
```

**Pull request merged**